bin/proving-service/src/commands/mod.rs

```diff
@@ -44,22 +43,20 @@ pub(crate) struct ProxyConfig {
     /// aborted.
     #[clap(long, default_value = "100", env = "MPS_TIMEOUT_SECS")]
     pub(crate) timeout_secs: u64,
-    /// Worker update service port.
-    #[clap(long, default_value = "8083", env = "MPS_MPS_WORKERS_UPDATE_PORT")]
-    pub(crate) workers_update_port: u16,
+    /// Control port.
+    #[clap(long, default_value = "8083", env = "MPS_CONTROL_PORT")]
+    pub(crate) control_port: u16,
 }
 
-#[derive(Debug, Parser)]
-pub(crate) struct MetricsConfig {
-    /// Enable metrics.
-    #[clap(long, default_value = "false", env = "MPS_ENABLE_METRICS")]
-    pub(crate) enable_metrics: bool,
-    /// Prometheus metrics host.
-    #[clap(long, default_value = "0.0.0.0", env = "MPS_PROMETHEUS_HOST")]
-    pub(crate) prometheus_host: String,
-    /// Prometheus metrics port.
-    #[clap(long, default_value = "9090", env = "MPS_PROMETHEUS_PORT")]
-    pub(crate) prometheus_port: u16,
+#[derive(Debug, Clone, clap::Parser)]
+pub struct MetricsConfig {
+    /// Enable Prometheus-compatible metrics
+    #[arg(long, env = "MPS_ENABLE_METRICS", default_value = "false")]
+    pub enable_metrics: bool,
+
+    /// Port for Prometheus-compatible metrics
+    #[arg(long, env = "MPS_METRICS_PORT", default_value = "6192")]
+    pub metrics_port: u16,
```

Comment:
Question: do we actually need the separate `enable_metrics` option? Could we not make `metrics_port` just an optional parameter and then assume that if it is not provided, metrics are not on?

Reply:
Makes sense 👌🏼 I'm changing it.